### DISCUSSION TITLE
learn-rust/

### DISCUSSION THREAD

**Discussion by giscus** [Dhravya/blog#11] [General]
# learn-rust/

My thoughts on Rust, learning Rust part 1

https://blog.dhravya.dev/learn-rust/

**Comment by nexxeln** [comment]
# Sheesh

**Reply by Dhravya** [reply]
# Sheesh

**Comment by saumsy** [comment]
demn.. noice!

**Comment by yxshv** [comment]
Wow

**Reply by yxshv** [reply]
I think it’s a weird comment lol
